src/main/java/jenkins/plugins/office365connector/Office365ConnectorWebhookNotifier.java

```diff
@@ -129,7 +130,19 @@ public void sendBuildMessage(StepParameters stepParameters) {
 
         WebhookJobProperty property = (WebhookJobProperty) run.getParent().getProperty(WebhookJobProperty.class);
         if (property == null) {
-            return;
+            String webhookUrl = stepParameters.getWebhookUrl();
```

Comment:
PR #19 accidental removed our way of using the plugin.
Though happily it gave me a chance to rewrite it :)